graphql/schema/types/task.graphql

```diff
@@ -76,6 +76,7 @@ type Task {
   projectId: String!
   projectIdentifier: String
   requester: String!
+  resetWhenFinished: Boolean!
```

Comment:
It could be nice to test for this field in a small GraphQL test

Reply:
I had intentionally omitted writing a test for this field. Since the field has no underlying logic/resolver and is just inherited from an `apiTask` it would panic if it was not defined.